### ISSUE TITLE
More optimisation for tablet style.

### ISSUE THREAD

**Issue by Thewisem** [florisboard/florisboard#962]
I eould like to point out that I feel like your keyboard is meant for a portrait keyboard. When I use it on my tab, it feels like an strech of a small device keyboard to a bigger device.
There are button changes and a lot of changes in a tablet oriented keyboard like this.

![Screenshot_20210529-222515](https://user-images.githubusercontent.com/74511929/120082648-beb00480-c0cc-11eb-9adc-dc62238964b8.jpg)
And this is what Florisboard looks like out of the box (with minimal optimisation)
![Screenshot_20210529-222606](https://user-images.githubusercontent.com/74511929/120082667-dedfc380-c0cc-11eb-9b96-9aa53b001990.jpg)
To not be biased, both are under same configuration. Its just one feels more natural and the other feels streched out. Sorry if this is insulting.

**Comment by Glitchy-Tozier** [comment]
I agree that this is interesting. I know people that type on tablets, just like you would do on a physical keyboard.

The seem really happy to have a control-key they can use for CTRL+C and  CTRL+V. (I don't understand how you could use a tablet this way but apparently it can be handy)

I can share a screenshot of how the Samsung-keyboard looks in ~30h.

**Comment by Thewisem** [comment]
So? What is the progress?

**Comment by Glitchy-Tozier** [comment]
Currently the focus for the next two updates will be on

1. Internal stuff and settings
2. Improving word suggestions and adding languages.

Tablet will probably have to wait a few more months, but I can't give you an exact timeline.

**Comment by patrickgold** [comment]
> So? What is the progress?

As Glitchy mentioned, the current focus is reworking the preference core+UI and improving word suggestions after that. Tablet optimizations may be done in-between if it fits in with the development of settings, but it is planned to be done after those two big projects.
